test/infamy/topology.py

```diff
@@ -6,15 +6,28 @@ def qstrip(text):
         return text[1:-1]
     return text
 
-def match_kind(n1attrs, n2attrs):
+def match_node(n1attrs, n2attrs):
     return n1attrs.get("kind") == n2attrs.get("kind")
 
+def match_edge(n1attrs, n2attrs):
```

Comment:
`n1attrs` and `n2attrs` should probably be `e1attrs` and `e2attrs` since we are comparing _edge_ attributes.